tests/unit/rankers/test_algorithms.py

```diff
@@ -254,13 +254,14 @@ def test_invalid_trailing_period(self):
                 trailing_period=123  # Model does not have 123 day trailing metrics
             )
 
+    @pytest.mark.parametrize("repeat", range(10))  # Increase 100x when changing thompson_sampling to avoid a flaky test
```

Comment:
Repeating this test 10 times might help to distinguish a flaky test from an actual bug, if it fails 1/10 times. We also have an `ntrials` parameter. I think they serve different purposes?